content/en/boilerplates/gateway-api-choose.md

```diff
@@ -1,5 +1,5 @@
 ---
 ---
 The following instructions allow you to choose to use either the Gateway API or the Istio configuration API when configuring
-traffic management in the mesh. Follow instructions under either the `Gateway API` or `Istio classic` tab,
+traffic management in the mesh. Follow instructions under either the `Gateway API` or the `Istio APIs` tab,
```

Comment:
Is this grammatically correct? I think this could be `either the "A" or "B" tab` or `either the "A tab" or the "B tab"`, but `either the "A" or the "B tab"` seems awkward?

Reply:
Hmm @ericvn's suggested this specific formation, but I agree it feels awkward. Will change it back for now and ask for Eric's input in this comment thread